erpnext/crm/doctype/lead/lead.json

```diff
@@ -209,10 +215,10 @@
    "remember_last_selected_value": 0, 
    "report_hide": 0, 
    "reqd": 0, 
-   "search_index": 1, 
+   "search_index": 0, 
    "set_only_once": 0, 
    "translatable": 0, 
-   "unique": 0
+   "unique": 1
```

Comment:
It will break for many users while setting unique in db. There are many duplicate email ids as the validations was missing at the beginning.

Reply:
Can we do anything to not affect existing records?

Reply:
I can't think any solution. Closing the PR.